src/info/LiquidatorInfo.sol

```diff
@@ -127,16 +127,29 @@ contract LiquidatorInfo is Math {
     function getCushionInfo(uint cdp, address me, uint numMembers) public view returns(CushionInfo memory info) {
         CdpDataVars memory c;
         (c.cdpArt, c.cushion, c.cdpWinners, c.bite) = pool.getCdpData(cdp);
-        info.isToppedUp = c.cushion > 0;
+        info.isToppedUp = manager.cushion(cdp) > 0;
         bool isUntoppedByUser = manager.cushion(cdp) == 0;
 
+        bool mustCallUntop = c.cushion > 0 && isUntoppedByUser;
+
+        if(info.isToppedUp && isUntoppedByUser || mustCallUntop) {
```

Comment:
`info.isToppedUp && isUntoppedByUser` will always return `false` imo.
The variable `isUntoppedByUser` is misleading, so might as well just check:
`if(c.cushion > 0 && manager.cushion(cdp) == 0)`